test/rotoscope_test.rb

```diff
@@ -288,6 +288,22 @@ def test_trace_ignores_calls_if_blacklisted
     assert_frames_consistent contents
   end
 
+  def test_trace_saves_called_context_if_called_from_blacklisted_path
+    dependency = FixtureInner.new
+    contents = rotoscope_trace(blacklist: ['/rotoscope_test.rb']) do
+      dependency.calls_sum
+    end
+
+    assert_equal [
+      {:event=>"call", :entity=>"FixtureInner", :method_name=>"calls_sum", :method_level=>"instance", :filepath=>"/rotoscope_test.rb", :lineno=>-1},
```

Comment:
Should this event be logged?  Since the call is from a blacklisted path.  If so, then are you proposing that we blacklist based on the filepath of the called method rather than from where the call took place?

Reply:
Well, the idea here is that we want to know that `FixtureInner#calls_sum` calls the `sum` method, even though `FixtureInner#calls_sum` was itself called from a blacklisted context. So, I think this should be logged, because otherwise we don't have the context to know how the `sum` method was called.

Reply:
I opened https://github.com/Shopify/rotoscope/pull/51 to fix the caller in the flattened output.  Although, I guess that change would make the flattened output not derivable from the non-flattened output.